.github/workflows/js-build.yml

```diff
@@ -39,6 +39,8 @@ jobs:
           EXPECTED_WALLET_ACCOUNT_ETH_ADDRESS: ${{ secrets.EXPECTED_WALLET_ACCOUNT_ETH_ADDRESS }}
           BANNED_TO_ADDRESS: "0x6F72eDB2429820c2A0606a9FC3cA364f5E9b2375"
           SOLANA_TEST_ORG_API_PRIVATE_KEY: ${{ secrets.SOLANA_TEST_ORG_API_PRIVATE_KEY }}
+          PUBLIC_RPC_URL: ${{ secrets.PUBLIC_RPC_URL }}
```

Comment:
I think we could leave this public rpc url within the test suite itself (unless we want to use a private RPC with a specific/paid API key)

Reply:
@andrewkmin and just use the community one?
`https://sepolia.infura.io/v3/84842078b09946638c03157f83405213`
if so, should I remove from here and secrets in github?